package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "celebrate",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
Please don't bump the package version. That typically happens in a separate commit, and should be done by the maintainers.